website/docs/developer/deployment-guide/creating-orca-glacier-bucket.md

```diff
@@ -192,11 +196,69 @@ aws sts get-caller-identity
 }
 ```
 
-Replace the value of `arn:aws:iam::909121343565:root` with the value of your
+Replace the number in `arn:aws:iam::909121343565:root` with the value of your
 account number.
 
 The Resource value is the bucket and bucket paths that the Cumulus application
-can access. Replace `arn:aws:s3:::sandbox-orca-glacier-archive` with the name
+can access. Replace `sandbox-orca-glacier-archive` with the name
 of the Orca archive bucket created in the previous section.
 
+##### Reports Bucket:
+
+The policy shown below can be used with some minor
+modifications, which will be detailed below.
 
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "Cross Account Access",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "arn:aws:iam::909121343565:root"
+      },
+      "Action": [
+        "s3:GetObject*",
+        "s3:GetBucket*",
+        "s3:ListBucket",
+        "s3:PutObject",
+        "s3:PutObjectAcl"
```

Comment:
Is "s3:PutObject", "s3:PutObjectAcl" needed since this is for writes on cross account? The report bucket should only need reads on the cross account (i.e. Database reading from S3 bucket.).

Reply:
Sadly, it is. We have to copy reports to add the metadata tag that AWS neglects.

Reply:
Ahh, forgot about that. Thank you for reminding me.